pom.xml

```diff
@@ -90,6 +90,16 @@
                 <checksumPolicy>ignore</checksumPolicy>
             </releases>
         </repository>
+        <repository>
+            <id>nuiton</id>
+            <name>nuiton Repository</name>
+            <url>http://maven.nuiton.org/release/</url>
```

Comment:
Is this new repo addition required for some newly added dependency?

Reply:
yes, this is for below dependency
<dependency>
                <groupId>com.sun</groupId>
                <artifactId>tools</artifactId>
                <version>${com.sun.tools.version}</version>
            </dependency>

this is available in the jdk installation as well, we can get that from jdk directory(via systempath), but that location is platform dependent, (we can use profiles to change system path according to the platform too) but all this seems extra effort  :) adding this repo simply resolves the problem.

Reply:
ack

Reply:
Looks like this will not work on all the OS. On Mac, Im seeing a ClassDefNotFound error with this PR

```
[ERROR] cleanup(org.ballerinalang.test.data.streaming.TableToXMLStreamingTestCase)  Time elapsed: 52.882 s  <<< FAILURE!
java.lang.NoClassDefFoundError: Could not initialize class sun.tools.attach.LinuxVirtualMachine
	at org.ballerinalang.test.data.streaming.TableToXMLStreamingTestCase.cleanup(TableToXMLStreamingTestCase.java:74)

[ERROR] destroy(org.ballerinalang.test.data.streaming.TableToXMLStreamingTestCase)  Time elapsed: 52.955 s  <<< FAILURE!
java.lang.NoClassDefFoundError: Could not initialize class sun.tools.attach.LinuxVirtualMachine

[ERROR] clean(org.ballerinalang.test.data.streaming.TableToXMLStreamingTestCase)  Time elapsed: 53.065 s  <<< FAILURE!
java.lang.NoClassDefFoundError: Could not initialize class sun.tools.attach.LinuxVirtualMachine
```